keras/src/optimizers/muon.py

```diff
@@ -53,7 +53,7 @@ class Muon(optimizer.Optimizer):
             (in the formula just before Section 2.1),
             not the epsilon in Algorithm 1 of the paper.
             It be used at Adamw.Defaults to `1e-7`.
-        exclude_layers: List of strings, keywords of layer names to exclude.
+        exclude_layers: str, keywords of layer names to exclude.
```

Comment:
I believe we should change the *behavior* of the argument to make it a list of strs. The most intuitive usage would be to pass the list of layer names you want to exclude.